languages/python/sqlalchemy-oso/requirements.txt

```diff
@@ -1,4 +1,3 @@
 oso==0.20.0-beta
 SQLAlchemy>=1.3.17,<1.5
 packaging~=20.9
-dataclasses; python_version<"3.7"
```

Comment:
😱

Reply:
not hurtin' anything, but no reason not to remove it now